lib/CleantalkSP/SpbctWP/ListTable.php

```diff
@@ -610,6 +623,27 @@ public static function ajaxRowActionHandlerDisapprove()
         wp_send_json($out);
     }
 
+    /**
+     * Ajax wrapper to approve or disapprove a page.
+     * @params string $action  Could be 'approve' or 'disapprove'
+     * @throws \Exception if internal handler failed
+     */
+    public static function ajaxRowActionHandlerPageApprovementProcess($action)
+    {
+        // handle page, throws exception on fail
+        spbc_scanner_page_approve_process($action);
```

Comment:
Wrap this by try...catch.

Reply:
I was wrong. The exceptions catchin doing here https://github.com/CleanTalk/security-malware-firewall/pull/205/files#diff-404405e00e28d72e5b4d80d3d8509b0f94477f01eed1e23885caacc474910275R543
Closing